object_detection/.gitignore

```diff
@@ -0,0 +1,2 @@
+faster_rcnn_nas_coco_2018_01_28
+ssdlite_mobilenet_v2_coco_2018_05_09
```

Comment:
There is no need to have multiple `.gitignore` files, let's use the one in the root of the directory only.

Also, all the models should be inside a folder: `object_detection/models` this way we can ignore that folder directly, so you can remove this file and add the following line to the root .gitignore `object_detection/models`

Reply:
yeah it make sense. So i think i will
- remove the unnecessary .gitignore 
- create **object_detection/models** and put *models/* into  .gitignore
-  Fix the code so that it can work with new path of models

Reply:
Yes, that would be perfect!

Reply:
ok